packages/launchpad/src/setup/FrameworkOptionsFooter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="flex bg-gray-50 h-65px py-12px px-17px text-gray-800 items-center">
```

Comment:
Do you mind trying getting the height of this down to: `h-48px`?

It feels a bit tall at the moment and almost has more visual weight than the links above it so I'm just worried about this new link competing with the others.

Reply:
Updated